operational/flexera/custom_branding/flexera_custom_branding.pt

```diff
@@ -0,0 +1,359 @@
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  category "Policy Settings"
+  label "Email Addresses"
+  description "A list of email addresses to notify if bill processing errors are found."
+  default []
+end
+parameter "param_nav_logo" do
+  type "string"
+  category "All Customer Settings"
+  label "Nav Logo Image"
+  description "The web address of the image resource file location to use in place of the Flexera logo that displays in the app navigation’s expanded state."
+  default "No Change"
+end
+
+parameter "param_nav_logo_small" do
+  type "string"
+  category "All Customer Settings"
+  label "Nav Logo Small image"
+  description "The web address of the image resource file location to use in place of the small Flexera logo that displays in the app navigation’s collapsed state."
+  default "No Change"
+end
+
+parameter "param_marketing_footer_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Footer Hide"
+  description "A boolean to determine show/hide of the marketing footer on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_marketing_submit_ideas_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Submit Ideas Hide"
+  description "A boolean to determine show/hide of the Submit Ideas link on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_marketing_open_new_case_link" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Open New Case Link"
+  description "A URL override pointing to your own Open New Case link on the landing page."
+  default "No Change"
+end
+
+parameter "param_marketing_support_help_link" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Support Help Link"
+  description "A URL override pointing to your own Support Help link on the landing page."
+  default "No Change"
+end
+
+parameter "param_marketing_support_plan_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Support Plan Hide"
+  description "A boolean to determine show/hide of the Support Plan Information link on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_marketing_flexera_com_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Submit Ideas Hide"
+  description "A boolean to determine show/hide of the Flexera.com link on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_login_logo_image" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Login Logo Image"
+  description "The web address of the image resource file location to use in place of the Flexera logo for unauthenticated pages like login, password reset, and so on."
+  default "No Change"

```

Comment:
Typically, if the parameter is asking for optional information, we should set the default to an empty string and put something in the description like "Leave blank to not configure a Nav Logo Image."